src/software/backend/input/network/filter/ball_filter_test.cpp

```diff
@@ -240,6 +240,9 @@ class BallFilterTest : public ::testing::Test
     Duration time_step;
     std::mt19937 random_generator;
     Timestamp current_timestamp;
+    // For these tests, the ball is always on the ground. The filters
+    // are not designed for filtering balls in the air
+    static constexpr double BALL_HEIGHT_METRES = 0.0;
```

Comment:
Could we rename to `BALL_DISTANCE_FROM_GROUND`, generally I would measure the height of the ball from it's center.

Reply:
done